workbench-core/infrastructure/src/workbenchAssociateStackToExistingAppRegistryApplication.ts

```diff
@@ -31,5 +32,6 @@ export class WorkbenchAssociateStackToExistingAppRegistryApplication extends Con
     );
 
     importedApplication.associateApplicationWithStack(stack);
+    createAppInsightsConfiguration(stack);
```

Comment:
Is there any limitation of the amount of cfn applications we can create or associate to a stack ?